src/controllers/leader-board.controller.ts

```diff
@@ -4,8 +4,8 @@ import {param, get, getModelSchemaRef, response} from '@loopback/rest';
 import {PaginationInterceptor} from '../interceptors';
 import {LeaderBoard} from '../models';
 import {LeaderBoardRepository} from '../repositories';
-import {authenticate} from '@loopback/authentication';
 import {LoggingBindings, logInvocation, WinstonLogger} from '@loopback/logging';
+import {authenticate} from '@loopback/authentication';
```

Comment:
leaderboard is public access, i think we need GET endpoint without auth

Reply:
okay sure